Test/Mono.Cecil.Tests/PortablePdbTests.cs

```diff
@@ -666,6 +666,22 @@ public void TypeDefinitionDebugInformation ()
 			}, symbolReaderProvider: typeof (PortablePdbReaderProvider), symbolWriterProvider: typeof (PortablePdbWriterProvider));
 		}
 
+		[Test]
+		public void ModifyTypeDefinitionDebugInformation ()
+		{
+			using (var module = GetResourceModule ("TypeDefinitionDebugInformation.dll", new ReaderParameters { SymbolReaderProvider = new PortablePdbReaderProvider () })) {
+				var enum_type = module.GetType ("TypeDefinitionDebugInformation.Enum");
+				var binary_custom_debug_info = enum_type.CustomDebugInformations.OfType<BinaryCustomDebugInformation> ().FirstOrDefault ();
+				binary_custom_debug_info.Data = new byte [] { 0x2 };
```

Comment:
nit: just to make it clear that it's different data before without having to look elsewhere

```suggestion
				Assert.AreEqual (new byte [] { 0x1 }, binary_custom_debug_info.Data);
				binary_custom_debug_info.Data = new byte [] { 0x2 };
```